src/client-agent/main.c

```diff
@@ -164,6 +164,9 @@ int main(int argc, char **argv)
         merror_exit(USER_ERROR, user, group);
     }
 
+    /* Check client keys */
+    OS_ReadKeys(&keys, 1, 0, 0);
```

Comment:
If we call this function here, we should avoid to call it again in AgentdStart() some lines below.

It also has to have the same behavior on Windows.

Reply:
Good catch, I didn't see that! Thanks!